### PR TITLE
fix: M1 and non-M1 macs now agree on xcodeproj

### PR DESCRIPTION
**Pull request by pvinis**
This PR resolves [] 

### Description

M1 macs and non-M1 macs generated different `EXCLUDED_ARCHS`, because of https://github.com/facebook/react-native/blob/0.66-stable/scripts/react_native_pods.rb#L135-L139. That is fixed and remove on RN 0.67, but we are still on 0.66, so I added that tiny change as a patch, so our macs can stop fighting about what needs to be in that variable.

### PR Checklist

- [ ] I tested my changes on **iOS** / **Android**.
- [ ] I added screenshots or videos to illustrate my changes.
- [ ] I added Tests and Stories for my changes.
- [ ] I added an [app state migration].
- [ ] I hid my changes behind a [feature flag].
- [ ] I have prefixed changes that need to be tested during a release QA with **[NEEDS EXTERNAL QA]** on the changelog.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to run this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

-

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

### CODE REVIEW

**Comment by gkartalis** [comment]
Works in an m1, builds the app without generating the diff anymore 🤘

**Review by gkartalis — APPROVED** [review]

**Review by MrSltun — APPROVED** [review]
Works in a non-m1 and builds the app without any diff

**Comment by pvinis** [comment]
awesome! now to figure out what's wrong with the tests. do you both mind hitting cmd-u on Xcode?

**Comment by gkartalis** [comment]
> awesome! now to figure out what's wrong with the tests. do you both mind hitting cmd-u on Xcode?

I missed this comment, happy to try it out

File: ios/Artsy.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -4273,7 +4290,6 @@
 				CB43287327E8FEE500B29DD6 /* ARValueTransformerTests.m in Sources */,
 				CB4328BB27E8FEE500B29DD6 /* ArtsyAPI+PrivateTests.m in Sources */,
 				CB43287427E8FEE500B29DD6 /* ARFileUtilsTests.m in Sources */,
-				CB43287B27E8FEE500B29DD6 /* SaleOnHoldOverlayViewTests.swift in Sources */,
```
Comment (pvinis) [inline ios/Artsy.xcodeproj/project.pbxproj:4276]:
```
				CB43287B27E8FEE500B29DD6 /* SaleOnHoldOverlayViewTests.swift in Sources */,
```

File: ios/Artsy.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -4273,7 +4290,6 @@
 				CB43287327E8FEE500B29DD6 /* ARValueTransformerTests.m in Sources */,
 				CB4328BB27E8FEE500B29DD6 /* ArtsyAPI+PrivateTests.m in Sources */,
 				CB43287427E8FEE500B29DD6 /* ARFileUtilsTests.m in Sources */,
```
Comment (pvinis) [inline ios/Artsy.xcodeproj/project.pbxproj:4292]:
```suggestion
				CB43287427E8FEE500B29DD6 /* ARFileUtilsTests.m in Sources */,
				CB43287B27E8FEE500B29DD6 /* SaleOnHoldOverlayViewTests.swift in Sources */,
```

File: ios/Artsy.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -113,7 +118,6 @@
 		CB43287727E8FEE500B29DD6 /* ARAugmentedRealityConfigSpec.m in Sources */ = {isa = PBXBuildFile; fileRef = CB43273C27E8FEE300B29DD6 /* ARAugmentedRealityConfigSpec.m */; };
 		CB43287927E8FEE500B29DD6 /* ARSerifNavigationViewControllerSpec.m in Sources */ = {isa = PBXBuildFile; fileRef = CB43273F27E8FEE300B29DD6 /* ARSerifNavigationViewControllerSpec.m */; };
 		CB43287A27E8FEE500B29DD6 /* LiveAuctionCurrentLotCTAPositionManagerTests.swift in Sources */ = {isa = PBXBuildFile; fileRef = CB43274027E8FEE300B29DD6 /* LiveAuctionCurrentLotCTAPositionManagerTests.swift */; };
```
Comment (pvinis) [inline ios/Artsy.xcodeproj/project.pbxproj:120]:
```suggestion
		CB43287A27E8FEE500B29DD6 /* LiveAuctionCurrentLotCTAPositionManagerTests.swift in Sources */ = {isa = PBXBuildFile; fileRef = CB43274027E8FEE300B29DD6 /* LiveAuctionCurrentLotCTAPositionManagerTests.swift */; };
		CB43287B27E8FEE500B29DD6 /* SaleOnHoldOverlayViewTests.swift in Sources */ = {isa = PBXBuildFile; fileRef = CB43274127E8FEE300B29DD6 /* SaleOnHoldOverlayViewTests.swift */; };
```

File: ios/Artsy.xcodeproj/project.pbxproj [hunk 4]
```diff
@@ -703,7 +713,6 @@
 		CB43273D27E8FEE300B29DD6 /* greyscaleImage.jpg */ = {isa = PBXFileReference; lastKnownFileType = image.jpeg; path = greyscaleImage.jpg; sourceTree = "<group>"; };
 		CB43273F27E8FEE300B29DD6 /* ARSerifNavigationViewControllerSpec.m */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.c.objc; path = ARSerifNavigationViewControllerSpec.m; sourceTree = "<group>"; };
 		CB43274027E8FEE300B29DD6 /* LiveAuctionCurrentLotCTAPositionManagerTests.swift */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.swift; path = LiveAuctionCurrentLotCTAPositionManagerTests.swift; sourceTree = "<group>"; };
```
Comment (pvinis) [inline ios/Artsy.xcodeproj/project.pbxproj:715]:
```suggestion
		CB43274027E8FEE300B29DD6 /* LiveAuctionCurrentLotCTAPositionManagerTests.swift */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.swift; path = LiveAuctionCurrentLotCTAPositionManagerTests.swift; sourceTree = "<group>"; };
		CB43274127E8FEE300B29DD6 /* SaleOnHoldOverlayViewTests.swift */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.swift; path = SaleOnHoldOverlayViewTests.swift; sourceTree = "<group>"; };
```

File: ios/Artsy.xcodeproj/project.pbxproj [hunk 5]
```diff
@@ -1812,7 +1826,6 @@
 				CB43273927E8FEE300B29DD6 /* ARVIR */,
 				CB43273E27E8FEE300B29DD6 /* Components */,
 				CB43274027E8FEE300B29DD6 /* LiveAuctionCurrentLotCTAPositionManagerTests.swift */,
```
Comment (pvinis) [inline ios/Artsy.xcodeproj/project.pbxproj:1828]:
```suggestion
				CB43274027E8FEE300B29DD6 /* LiveAuctionCurrentLotCTAPositionManagerTests.swift */,
				CB43274127E8FEE300B29DD6 /* SaleOnHoldOverlayViewTests.swift */,
```

**Pull request merged**